packages/web/pages/settings/saved-searches.tsx

```diff
@@ -349,7 +352,7 @@ export default function SavedSearchesPage(): JSX.Element {
               <Box>
                 <StyledText style="fixedHeadline">Saved Searches </StyledText>
               </Box>
-              <InfoLink href="https://docs.omnivore.app/using/search.html" />
+              <InfoLink href="/help/search" />
```

Comment:
I think maybe you need a rebase?

Reply:
Merged the wrong thing here I think, will fix